cmd/utils/flags.go

```diff
@@ -742,6 +742,11 @@ var (
 		Usage: "Set the value transfer recovery interval (seconds)",
 		Value: 60,
 	}
+	DefaultTxGasLimitFlag = cli.Uint64Flag{
+		Name:  "defaultgaslimit",
+		Usage: "Set the default value of gas limit for transactions made by bridges",
+		Value: 5000000,
```

Comment:
Just out of curiosity, where does this value come from?

Reply:
https://github.com/klaytn/klaytn/pull/1287/files#diff-0f905cce4ddf20349092d51aad72ea452c5172650106b834c6bab18f4ce4d2b6
Originally, `DefaultBridgeTxGasLimit` was set to 5000000, so I set this default flag value to 5000000, too.